README.md

```diff
@@ -1,41 +1,35 @@
 ﻿# react-native-share [![CircleCI](https://circleci.com/gh/react-native-community/react-native-share/tree/master.svg?style=svg&circle-token=0c6860240abba4e16bd07df0ea805a72b67b8d41)](https://circleci.com/gh/react-native-community/react-native-share/tree/master) [![npm version](https://badge.fury.io/js/react-native-share.svg)](http://badge.fury.io/js/react-native-share)
 Share Social , Sending Simple Data to Other Apps
 
-***NOTE: React Native now implements share functionality [Read more](https://facebook.github.io/react-native/docs/share.html)***
-
-## Sponsors
+# Sponsors
+---
 If you use this library on your commercial/personal projects, you can help us by funding the work on specific issues that you choose by using IssueHunt.io!
 
 This gives you the power to prioritize our work and support the project contributors. Moreover it'll guarantee the project will be updated and maintained in the long run.
 
 [![issuehunt-image](https://issuehunt.io/static/embed/issuehunt-button-v1.svg)](https://issuehunt.io/repos/43406976)
 
-## Getting started
-
-### Mostly automatic install
-1. `yarn add react-native-share`
-2. `react-native link react-native-share`
 
-#### How to use master branch?
+# Getting started
+---
 
-We recommend using the releases from npm, however you can use the master branch if you need any feature that is not available on NPM. By doing this you will be able to use unreleased features, but the module may be less stable. 
+## Automatic Way
 
-**yarn**: `yarn add react-native-share@git+https://git@github.com/react-native-community/react-native-share.git`
 
-**npm**: `npm install --save react-native-share@git+https://git@github.com/react-native-community/react-native-share.git`
-
-### Manual install
+```sh 
+yarn add react-native-share
+react-native link react-native-share
+```
 
-`npm install react-native-share --save`
+We recommend using the releases from npm, however you can use the master branch if you need any feature that is not available on NPM. By doing this you will be able to use unreleased features, but the module may be less stable. 
+**yarn**: ```sh yarn add react-native-share@git+https://git@github.com/react-native-community/react-native-share.git```
```

Comment:
remove sh

Reply:
> remove sh

Sure, sh was there to tell the script was need to open on shell or command line. I will update this to comment.